apps/studio/components/interfaces/Settings/Logs/LogsQueryPanel.tsx

```diff
@@ -14,28 +7,39 @@ import {
   DropdownMenuContent,
   DropdownMenuItem,
   DropdownMenuTrigger,
-  IconBookOpen,
   IconCheck,
-  IconChevronDown,
   IconClipboard,
   IconExternalLink,
   IconX,
   Popover,
   SidePanel,
   Tabs,
 } from 'ui'
+
+import Link from 'next/link'
```

Comment:
nit: just fyi for consistency we're following a writing style where we separate our import statements as such:

```
// Example:
<external imports>
import Link from 'next/link'

<internal imports>
import DatePickers from './Logs.DatePickers'
```